rskj-core/src/main/java/co/rsk/net/SnapshotProcessor.java

```diff
@@ -56,47 +66,41 @@
  */
 public class SnapshotProcessor {
 
-    private static final Logger logger = LoggerFactory.getLogger("snapshotprocessor");
-
     public static final int BLOCK_NUMBER_CHECKPOINT = 5000;
     public static final int BLOCK_CHUNK_SIZE = 400;
     public static final int BLOCKS_REQUIRED = 6000;
+    private static final Logger logger = LoggerFactory.getLogger("snapshotprocessor");
```

Comment:
I would move this to were it was as all the majority of our files have the logger at first. WDYT?

Reply:
Oh that was the formatter, I'll change it back because the formatter moved the logger to the bottom of the attributes

Reply:
solved!